src/main/java/me/jellysquid/mods/sodium/client/gl/buffer/GlBufferStorageFlags.java

```diff
@@ -5,6 +5,8 @@
 import org.lwjgl.opengl.GL44C;
 
 public enum GlBufferStorageFlags implements EnumBit {
+    NONE(0),
```

Comment:
This is a bit field... just create an empty set, rather than defining NONE here.

Reply:
Fixed, you cannot just create an empty bitfield as you have to pass in an element to the underlying EnumSet so that it can obtain the class